chandra_aca/dark_model.py

```diff
@@ -0,0 +1,281 @@
+        scale_4c = DARK_SCALE_4C
+
+    return scale_4c ** ((t_ccd_ref - t_ccd) / 4.0)
+
+
+def get_dark_model(date, t_ccd):
+    """
+    Return the dark current model corresponding to ``date`` and ``t_ccd``.
+
+    :param date: date in any DateTime format
+    :param t_ccd: CCD temperature (deg C)
+
+    :returns: TBD
+    """
+    raise NotImplementedError()
+
+
+def get_dark_hist(date, t_ccd):
+    """
+    Return the dark current histogram corresponding to ``date`` and ``t_ccd``.
+
+    :param date: date in any DateTime format
+    :param t_ccd: CCD temperature (deg C)
+
+    :returns: bin_centers, bins, darkhist
+    """
+    pars = get_sbp_pars(date)
+    x = darkbins.bin_centers
+    y = smooth_broken_pow(pars, x)
+
+    scale = dark_temp_scale(-19, t_ccd)
+    xbins = darkbins.bins * scale
+    x = x * scale
+    return x, xbins, y
+
+
+def get_acq_success(date, t_ccd, mag):
+    """
+    Return probability of acquisition success for given date, temperature and mag.
+
+    Any of the inputs can be scalars or arrays, with the output being the result of
+    the broadcasted dimension of the inputs.
+
+    This is based on the dark model and acquisition success fitting presented
+    in the State of the ACA 2013 (sot/state_of_aca/guide_acq_stats)
+
+    :param date: Date(s) (scalar or np.ndarray)
+    :param t_ccd: CD temperature(s) (degC, scalar or np.ndarray)
+    :param mag: Star magnitude(s) (scalar or np.ndarray)
+
+    :returns: Acquisition success probability(s)
+    """
+    try:
+        date = DateTime(date).secs
+        zeros = np.atleast_1d(np.zeros_like(date + mag + t_ccd))
+        dates = date + zeros
+        t_ccds = t_ccd + zeros
+        mags = mag + zeros
+    except:
+        raise ValueError("Incompatible input shapes for 'date', 't_ccd', 'mag'")
+
+    warm_fracs = []
+    for sdate, stemp  in izip(dates, t_ccds):
+        warm_frac = get_warm_fracs(warm_threshold,
+                                   date=sdate, T_ccd=stemp)
+        warm_fracs.append(warm_frac)
+    probs = acq_success_prob(mags, warm_fracs)
+
+    if (np.array(date).ndim == 0 and np.array(t_ccd).ndim == 0
+        and np.array(mag).ndim == 0):
+           return probs[0]
+    else:
+        return probs
+
+
+
+def get_guide_success(date, t_ccd, mag):
+    """
+    Return probability of guide (bad_trak) success for given date, temperature and mag.
+
+    Any of the inputs can be scalars or arrays, with the output being the result of
+    the broadcasted dimension of the inputs.
+
+    This is based on the dark model and guide success fitting presented
+    in the State of the ACA 2013 (sot/state_of_aca/guide_acq_stats)
+
+    :param date: Date(s) (scalar or np.ndarray)
+    :param t_ccd: CD temperature(s) (degC, scalar or np.ndarray)
+    :param mag: Star magnitude(s) (scalar or np.ndarray)
+
+    :returns: Guide success probability(s)
+    """
+    raise NotImplementedError()
+
+
+
+
+def acq_success_prob(mag, warm_frac, prob_offset=0):
```

Comment:
Can be deleted.  I checked our dependencies repos and this function is not used anywhere.  It has been superceded by the same-named function in `star_probs`.